src/Persistence.Tests/PaYaml/Serialization/PaYamlSerializerTests.cs

```diff
@@ -45,6 +45,22 @@ public void DeserializeExamplePaYamlApp(string path, int expectedAppPropertiesCo
         paFileRoot.App.Should().NotDefineMember("Children", "App.Children is still under design review");
     }
 
+    [TestMethod]
+    [DataRow(@"_TestData/SchemaV3_0/Examples/Src/DataSources/DataSources1.pa.yaml")]
```

Comment:
also add this file the round-trip test to ensure it can be round-tripped, not-lossy